themes/bootstrap3/templates/search/advanced/layout.phtml

```diff
@@ -105,8 +105,8 @@
                     <div id="new_search_template">
                   <?php endif; ?>
                   <div id="search<?=$group . '_' . $search ?>" class="adv-search">
-                    <input name="lookfor<?=$group ?>[]" id="search_lookfor<?=$group . '_' . $search ?>" class="adv-term-input form-control" type="text"<?php if (isset($setQueries[$group][$search])): ?> value="<?=$this->escapeHtml($setQueries[$group][$search]->getString())?>"<?php endif; ?>>
-                    <select class="adv-term-type form-control" name="type<?=$group ?>[]">
+                    <input name="lookfor<?=$group ?>[]" id="search_lookfor<?=$group . '_' . $search ?>" class="adv-term-input form-control" type="text"<?php if (isset($setQueries[$group][$search])): ?> value="<?=$this->escapeHtml($setQueries[$group][$search]->getString())?>"<?php endif; ?> aria-label="<?=$this->transEsc("search_terms")?> <?=$search ?>">
```

Comment:
Is including the $search number here actually necessary? Right now, it's not working -- because the Javascript works by duplicating elements from a template, when you add lines to the search group, they all get the same label of "Search type 0" / "Search terms 0". I think it would be much simpler to eliminate the number if we can get away with that. If we absolutely have to have non-unique labels, it will take some more work to get it working correctly and to use more appropriate translation strings with placeholders (instead of concatenating numbers onto labels).

Reply:
Lighthouse seems fine with the numbers being dropped, so I've removed them for now instead of trying to hack together something that will actually make them unique and translated usefully.